src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -633,7 +631,7 @@ def __call__(self):
             if funcName == "repack":
                 self.handleRepackSettings()
 
-            if funcName in ["repack", "merge", "alcaHarvesting" ]:
+            if funcName in ["merge", "alcaHarvesting" ]:
```

Comment:
Ok, I'll need to change your code then. There are merge and alcaHarvesting jobs that are multicore for HTCondopr, but should be singlecore for CMSSW.